app/src/main/res/layout/tab_fontes.xml

```diff
@@ -55,14 +55,27 @@
                         android:layout_height="wrap_content"
                         android:layout_centerVertical="true"
                         android:layout_gravity="center_vertical"
-                        android:layout_toStartOf="@+id/buttonListMachine"
+                        android:layout_toStartOf="@+id/buttonFilterListMachineFontes"
                         android:layout_toEndOf="@+id/imageMachine"
                         android:hint="@string/MachineHint"
                         android:imeOptions="actionNext"
                         android:inputType="text|textCapWords|textAutoComplete"
                         android:selectAllOnFocus="true"
                         android:singleLine="true" />
 
+                    <ImageButton
```

Comment:
I think that having an independant button for the filtering is not very intuitive. I would rather see the filtering directly  integrated into the List Dialog. However, I understand this make the development more complex as it requires creating a Custom Dialog. Tell me if that's a problem.

Reply:
I definitely get your point. I will try to solve that on my own. I already have an idea how that could be done.

Reply:
I have finished my implementation. I implemented a custom dialog by reusing the tab_machine.xml screen. Please have a look at it. I am looking forward to your feedback!

Reply:
Perfect. Thank you.